### PR TITLE
add proc_net_snmp6 charts info

### PR DESCRIPTION
**Pull request by ilyam8**
##### Summary

This PR adds dashboard info for [proc_net_snmp6](https://github.com/netdata/netdata/blob/master/collectors/proc.plugin/proc_net_snmp6.c) collector's charts.

##### Component Name

`web/`

##### Test Plan

Install this PR; check the proc_net_snmp6 charts; ensure the info is on the dashboard.

##### Additional Information

Fixes part of netdata/product#2104

### CODE REVIEW

File: web/gui/dashboard_info.js [hunk 1]
```diff
@@ -1590,6 +1590,164 @@ netdataDashboard.context = {
...
+        'Both these counters include errors.</p>'
+    },
+
+    'ipv6.icmpredir': {
+        info: 'The number of transferred ICMPv6 Redirect messages. '+
+        'These messages inform a host to update its routing information (to send packets on an alternative route).'
+    },
+
+    'ipv6.icmpechos': {
+        info: 'The number of ICMPv6 Echo messages.'
```
Comment (thiagoftsm) [inline web/gui/dashboard_info.js:1698]: Is `Echo` in uppercase expected? Or did we have typo error?

Comment (ilyam8) [reply web/gui/dashboard_info.js:1698]: Yes, all control name messages names start with an uppercase letter. See https://www.iana.org/assignments/icmpv6-parameters/icmpv6-parameters.xhtml

Comment (thiagoftsm) [reply web/gui/dashboard_info.js:1698]: Thank you!

**Review by thiagoftsm — APPROVED** [review]

File: web/gui/dashboard_info.js [hunk 2]
```diff
@@ -1590,6 +1590,164 @@ netdataDashboard.context = {
...
+    'ipv6.packets': {
+        info: '<p>IPv6 packet statistics for this host.</p>'+
+        '<p><b>Received</b> - packets received by the IP layer. '+
+        'This counter will be increased even if the packet is dropped later. '+
+        '<b>Sent</b> - packets sent via IP layer, for both single cast and multicast packets. '+
+        'This counter does not include any packets counted in <b>Forwarded</b>. '+
+        '<b>Forwarded</b> - input packets for which this host was not their final IP destination, '+
+        'as a result of which an attempt was made to find a route to forward them to that final destination. '+
+        'In hosts which do not act as IP Gateways, this counter will include only those packets which were '+
+        '<a href="https://en.wikipedia.org/wiki/Source_routing" target="_blank">Source-Routed</a>, '+
```
Comment (vlvkobal) [inline web/gui/dashboard_info.js:1605]:
```suggestion
        '<a href="https://en.wikipedia.org/wiki/Source_routing" target="_blank">Source-Routed</a> '+
```

File: web/gui/dashboard_info.js [hunk 3]
```diff
@@ -1590,6 +1590,164 @@ netdataDashboard.context = {
...
+        'to request that they advertise their presence on the network. '+
+        'Router <b>Advertisement</b> message is sent by a router on the local area network to announce its IP address '+
+        'as available for routing.</p>'
+    },
+
+    'ipv6.icmpneighbor': {
+        info: '<p>The number of transferred ICMPv6 '+
+        '<a href="https://en.wikipedia.org/wiki/Neighbor_Discovery_Protocol" target="_blank">Neighbour Discovery</a> messages.</p>'+
+        '<p>Neighbor <b>Solicitations</b> are used by nodes to determine the link layer address '+
+        'of a neighbor, or to verify that a neighbor is still reachable via a cached link layer address. '+
```
Comment (vlvkobal) [inline web/gui/dashboard_info.js:1730]:
```suggestion
        'of a neighbor or to verify that a neighbor is still reachable via a cached link layer address. '+
```

**Review by vlvkobal — APPROVED** [review]

**Pull request merged**